api/v2/models/integration.go

```diff
@@ -31,15 +31,15 @@ import (
 // swagger:model integration
 type Integration struct {
 
-	// last error
-	LastError string `json:"lastError,omitempty"`
-
-	// last notify
+	// last attempt
```

Comment:
please add more explanations. last attempt of what?

Reply:
This is a file generated by go-swagger. All client-facing structs follow the same pattern.